modules/gui/src/com/haulmont/cuba/gui/app/core/scheduled/messages.properties

```diff
@@ -31,6 +32,9 @@ finished=Finished
 
 methodParams=Method Parameters
 
+errorNotification.caption = Execution failed
+errorNotification.message = Your server is not on the permitted list
```

Comment:
Add information: how to fix this? Change permitted servers list or disable singleton ?